ruby/basics/arrays.md

```diff
@@ -34,28 +49,105 @@ Array.new(3, Array.new) #=> [[], [], []]
 ```
 
 ## Basic Methods
-Ruby gives you many methods to manipulate arrays and their contents, many of which are beyond the scope of this article. For full documentation, go to [http://ruby-doc.org/](http://ruby-doc.org/), click on "Core API", and scroll down to Classes: Array. There, you'll find the most up-to-date documentation on the various methods available Ruby arrays, along with explanations.
+Ruby gives you many methods to manipulate arrays and their contents, many of which are beyond the scope of this article. For full documentation, go to [http://ruby-doc.org/](http://ruby-doc.org/), click on "Core API", and scroll down to Classes: Array. There, you'll find the most up-to-date documentation on the various methods available to Ruby arrays, along with explanations.
 
-Calling `#methods` on an array will also yield a long list of methods, like so:
+Calling `#methods` on an array will also yield a long list of available methods, like so:
```

Comment:
I realize that `#methodName` is a standard way to refer to methods, but it could be confusing to a newbie. I'd suggest dropping the `#`. Since they've already done the Chris Pine lessons, they'll know to use the `object.method` convention. If not, they see it in your examples. And that way they aren't wondering "Where did the hashtag go?" or not fully reading the instructions and trying to call `object.#instruction`